src/scripts/release.zig

```diff
@@ -447,6 +447,7 @@ fn publish(shell: *Shell, languages: LanguageSet, info: VersionInfo) !void {
     }
 
     if (languages.contains(.zig)) {
+        try shell.project_root.setAsCwd();
```

Comment:
Should this be inside the node if statement instead?

Reply:
So far, functions don't generally try to not mess up with cwd --- rather each function sets the cwd it needs. 

I am thinking about a larger refactor which would enforce the invariant that cwd doesn't change from under your feet, but that's going to be a larger refactor :)

For the code as is, I think doing cd at this point is right --- `gh` is the thing that needs correct `cd`, so it's its job to enforce that.